tools/version_mapping.go

```diff
@@ -109,6 +109,9 @@ var (
 
 	// V25 Teep, builtin-actors(go-state-types): v16, calibration: 2523454, mainnet: 4878840
 	V25 version = version{calibration: 2523454, mainnet: buildconstants.UpgradeTeepHeight, nodeVersion: 25}
+
+	// V26 GoldenWeek, builtin-actors(go-state-types): v17, calibration: 3007294, mainnet: <unknown>
+	V26 version = version{calibration: 3007294, mainnet: buildconstants.UpgradeGoldenWeekHeight, nodeVersion: 26}
```

Comment:
What about this unknow here.... are we sure we are not going to start using this version, as the height value is unknow yet?